src/gui/presets.c

```diff
@@ -1154,9 +1154,12 @@ static void _menuitem_manage_quick_presets(GtkMenuItem *menuitem, gpointer data)
         const gboolean inlist = (config && strstr(config, txt));
         g_free(txt);
         gtk_tree_store_append(treestore, &child, &toplevel);
-        gtk_tree_store_set(treestore, &child, 0, presetname, 1, inlist, 2, TRUE, 3, g_strdup(iop->op), 4, g_strdup(name),
-                           -1);
+        gchar *dup_op = g_strdup(iop->op);
+        gchar *dup_name = g_strdup(name);
+        gtk_tree_store_set(treestore, &child, 0, presetname, 1, inlist, 2, TRUE, 3, dup_op, 4, dup_name, -1);
```

Comment:
can't we just remove g_strdup() here and just use iop->op & name in tree store?

Reply:
I hope so, but I'm not that familiar w/ GTK and its requirements...

Reply:
It cannot be otherwise to me anyway. We `g_strdup()`, pass to `gtk_tree_store_set()` and free the memory just after. So clearly the tree store does dup the string inside, and looking at the other calls in the code it is the case everywhere.